### PR TITLE
docs: formalized v1 maintenance docs

### LINKED ISSUE #7500: Docs: Formalize the v1 governance proposal into docs
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

Spinning this point out of https://github.com/typescript-eslint/typescript-eslint/discussions/7103:

> Formalize into the [`https://typescript-eslint.io/maintenance`](https://typescript-eslint.io/maintenance) docs

I'll send a PR to fill in any maintenance practices from the proposal into our docs.

### Affected URL(s)

https://typescript-eslint.io/maintenance

### LINKED ISSUE #7501: Docs: Add a 'team' page to the website
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

Spinning this point out of https://github.com/typescript-eslint/typescript-eslint/discussions/7103:

> Add a `typescript-eslint.io/team` page to the docs

I'll send a PR.

### Affected URL(s)

typescript-eslint.io/team

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## PR Checklist

- [x] Addresses an existing open issue: fixes #7500
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

Roughly copy & pastes the ratified governance proposal into our maintenance docs, significantly expanding that section of the website.

### CODE REVIEW

File: docs/maintenance/Pull_Requests.mdx [hunk 1]
```diff
@@ -13,6 +13,17 @@ It's imperative we give them a positive, uplifting experience.
 If you're ever unsure on any part of PR reviews, don't hesitate to loop in a maintainer that has more context to help!
 :::
 
+## Governance
+
+Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#pointing):
+
+- Straightforward: Aat reviewer discretion, may be merged with a single approval by any committer or maintainer.
```
Comment (JoshuaKGoldberg) [inline docs/maintenance/Pull_Requests.mdx:20]:
```suggestion
- Straightforward: At reviewer discretion, may be merged with a single approval by any committer or maintainer.
```

File: docs/maintenance/Issues.mdx [hunk 2]
```diff
@@ -13,6 +13,17 @@ It's imperative we give them a positive, uplifting experience.
...
 
+## Governance
+
+Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#pointing):
+
+- Straightforward: at reviewer discretion, may be marked as approved by any committer or maintainer.
+  This includes docs enhancements, bug fixes, and feature additions.
+- Non-straightforward: may be marked as approved with either two committer approvals or one maintainer approval.
+  These include significant internal refactors and non-breaking public API changes.
+- "Unusual"-categorized: require a public discussion followed by two maintainer approvals.
```
Comment (JoshuaKGoldberg) [inline docs/maintenance/Issues.mdx:24]: > `followed by two maintainer approvals`

Followup idea: maybe add committer here, for knowledge sharing?

File: docs/maintenance/Issues.mdx [hunk 3]
```diff
@@ -5,14 +5,25 @@ title: Issues
 
 This document serves as a guide for how you might manage our [GitHub Issues](https://docs.github.com/issues), also known as issue triaging.
 
-Use your best judgement when triaging issues, and most of all remember to be **kind, friendly, and encouraging** when responding to users.
+Use your best judgement when triaging issues, and most of all remember to be **encouraging, friendly, and kind** when responding to users.
```
Comment (JoshuaKGoldberg) [inline docs/maintenance/Issues.mdx:8]: Nit: alphabetizing a list 😄

**Review by bradzacher — DISMISSED** [review]

File: docs/maintenance/Issues.mdx [hunk 4]
```diff
@@ -21,17 +32,17 @@ Don't treat these as exact responses you must use: they're just a starting copy+
...
 
 1. A maintainer ensures the issue is valid:
    - If the poster didn't fill out an appropriate template with enough information:
-     - Add the `please fill out the template` and `awaiting response` labels
+     - Add the `please fill out the template` and `awaiting response` labels, and remove the `triage` label
      - Ask the poster for more information using a _Needs More Info_ response
    - If it's a duplicate of an issue that already exists:
-     - Add the `duplicate` label and remove the `bug` label
-     - If it's an obvious duplicate, post a _Clearly Duplicate Issue_ response
+     - Add the `duplicate` label and remove the `bug` and `triage` labels
```
Comment (bradzacher) [inline docs/maintenance/Issues.mdx:44]: update: don't remove the `bug` label I think?
makes it harder to find things.

**Pull request merged**